lte/gateway/c/core/oai/tasks/mme_app/CMakeLists.txt

```diff
@@ -36,11 +36,10 @@ add_library(TASK_MME_APP
     mme_app_sgs_null.c
     mme_app_sgs_la_updated.c
     mme_app_sgs_associated.c
-    mme_app_sgs_reset.c
+    mme_app_sgs_reset.cmme_app_sgsap.c
```

Comment:
File mme_app_sgsap.c is deleted, shall we remove from CMakeLists.txt file as well

Reply:
I'm not sure what happened here, thanks for the catch! I presume this doesn't compile right now (going to check CI results for `make build_oai`).

Reply:
Indeed the CI pipeline detected my error here (thankfully). Thank you for the review! I've fixed up the PR and pushed a revision.